README.md

```diff
@@ -536,6 +540,43 @@ We propose to give extensions the ability to select which partition key to use w
 It is worth noting that by allowing extension background contexts to load cookies across different partitions allows extensions to use partitioned cookies to store cross-site identifiers.
 This problem is discussed further in [Security and Privacy Considerations](#security-and-privacy-considerations).
 
+### First-Party CHIPS
+
+Typically, top-level site owners (i.e. the "first party") would want to use `SameSite=Lax/Strict` cookies on their own site, since these cookies have SameSite protections from click jacking and other forms of abuse.
```

Comment:
Wondering if we can keep this simple by replacing last bit with "..., since these cookies [offer CSRF protections](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-09#section-8.8).". The 6265 goes into more detail on the type of CSRF attacks that it SameSite protects against; and also references additional CSRF protection mechanisms that the site author should deploy.

Reply:
Done.